feedback-module/public/locales/en/translation.json

```diff
@@ -20,58 +20,49 @@
     ]
   },
 
-  "missingBtn": "I couldn't find what I was looking for",
-  "brokenBtn": "Something appears to be broken or inaccurate",
-  "otherBtn": "I havce other feedback on this page or website",
-
-  "missingTitle": "Sorry you couldn't find what you were looking for! Can you tell us a little more?",
-  "missingCheckboxes": [
-    "I wasn't sure where to look",
-    "I was confused by the information I found",
-    "I don't understand how to use this website",
-    "I hit a dead end",
-    "The content was not available in my preferred language",
-    "There were errors with the translation available",
-    "Other"
-  ],
-  "missingTextInputs": [
-    { "type": "text", "text": "What were you looking for or trying to do?" },
-    { "type": "text", "text": "Would you like to add anything else?" }
-  ],
-
-  "brokenTitle": "Sorry to hear that something is broken or inaccurate! Can you tell us a little more?",
-  "brokenCheckboxes": [
-    "There is a spelling mistake or typo",
-    "A link or button does not work",
-    "The information is out of date",
-    "I got different informatino from another source (311, printed mailer, elected official, etc.)",
-    "The layout looks broken",
-    "I got an error",
-    "Other"
-  ],
-  "brokenTextInputs": [
-    {
-      "type": "text",
-      "text": "Can you give us more details on what you were trying to do, or the exact problem you encountered? (optional)"
-    }
-  ],
+  "broken": {
+    "initialBtn": "Something appears to be broken or inaccurate",
+    "title": "Sorry to hear that something is broken or inaccurate! Can you tell us a little more?",
+    "checkboxes": [
+      "There is a spelling mistake or typo",
+      "A link or button does not work",
+      "The information is out of date",
+      "I got different informatino from another source (311, printed mailer, elected official, etc.)",
+      "The layout looks broken",
+      "I got an error",
+      "Other"
+    ],
+    "textInputs": [
+      {
+        "type": "text",
+        "text": "Can you give us more details on what you were trying to do, or the exact problem you encountered? (optional)"
+      }
+    ]
+  },
 
-  "otherTitle": "We appreciate your feedback.",
-  "otherTextInputs": [
-    { "type": "text", "text": "Can you tell us a little more?" }
-  ],
+  "other": {
+    "initialBtn": "I havce other feedback on this page or website",
+    "title": "We appreciate your feedback.",
+    "textInputs": [{ "type": "text", "text": "Can you tell us a little more?" }]
```

Comment:
Just a note for the future— I updated the types for some of the textInputs to textarea instead of text so that we don't need to create a separate size property to determine whether to use a Textarea or TextInput component in the generalized version of the screens & because the type wasn't being used as a prop in the Textarea components anyways.

Reply:
Ok sounds good, I can update that